ProcessList.c

```diff
@@ -199,7 +199,7 @@ static void ProcessList_removeIndex(ProcessList* this, const Process* p, int idx
    assert(Vector_countEquals(this->processes, Hashtable_count(this->processTable)));
 }
 
-static void ProcessList_buildTreeBranch(ProcessList* this, pid_t pid, int level, int indent, bool show) {
+static void ProcessList_buildTreeBranch(ProcessList* this, pid_t pid, unsigned int level, int32_t indent, bool show) {
```

Comment:
```suggestion
static void ProcessList_buildTreeBranch(ProcessList* this, pid_t pid, size_t level, int32_t indent, bool show) {
```

Reply:
Is the depth to be expected to be bigger than 2^32-1? (Then the Process member tree_depth also needs to be updated.)

Reply:
As the `indent` bit field is limited to 32 bits right now, we could even suffice to use `unit8_t` - I thought about that too, but decided to suggest the `size_t` change instead for the implicit "number of things" semantic that comes with it.